docs/utilities/data_classes.md

```diff
@@ -1121,3 +1120,28 @@ This example is based on the AWS Blog post [Introducing Amazon S3 Object Lambda
         for record in event.records:
             do_something_with(record.body)
     ```
+
+## Advanced
+
+### Debugging
+
+Alternatively, you can print out the fields to obtain more information. All classes come with a `__str__` method that generates a dictionary string which can be quite useful for debugging.
+
+However, certain events may contain sensitive fields such as `secret_access_key` and `session_token`, which are labeled as sensitive to prevent any accidental disclosure of confidential information.
+
+!!! warning "Some fields, like JSON dictionaries, can't be processed and will display as "[Cannot be deserialized]""
```

Comment:
Fields like JSON dictionaries can't be processed and will appear as `[Cannot be deserialized]`

Reply:
Thank you @heitorlessa and @neilramsay for the suggestions.

I made small changes and I think it is clear now.